doc/api/perf_hooks.md

```diff
@@ -98,6 +98,43 @@ added: v8.5.0
 Returns the current high resolution millisecond timestamp, where 0 represents
 the start of the current `node` process.
 
+### performance.startEventLoopTimer([options])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `resolution` {number} The sampling rate in milliseconds. Defaults to `10`.
+  * `reset` {boolean} `true` to reset the sampling data. Defaults to `false`.
+* Returns: {boolean} True if the timer was started. False if the timer has
+           already been started.
+
+Starts monitoring the event loop for delays. The accumulated event loop delay
+is reported as `performance.nodeTiming.eventLoopDelay`.
+
+The accumulated event loop delay (`delay`) is determined by:
+
+* Calculating a `threshold` in milliseconds as `5e6 + (resolution * 1e6)`.
+* For each turn of the event loop, determining the time in milliseconds elapsed
+  since the previous turn (the `delta`)
+* If the `delta` is greater than the `threshold`, add the `delta` to `delay`.
+
+For instance, given a `resolution` of 10 milliseconds, the delay threshold is
+`15000000` nanoseconds. If time of the previous turn exceeds this threshold,
+that time is added to the accumulated delay.
+
+### performance.stopEventLoopTimer([options])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `reset` {boolean} `true` to reset the sampling data. Defaults to `false`.
```

Comment:
why is there `reset` here?

Reply:
`reset: true` just zeroes out the counters. 

```js
startEventLoopTimer()
stopEventLoopTimer()
startEventLoopTimer({ reset: true })
stopEventLoopTimer()
```

is generally equivalent to...

```js
startEventLoopTimer()
stopEventLoopTimer({ reset: true })
startEventLoopTimer()
stopEventLoopTimer()
```